ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -300,6 +307,10 @@ object Main {
             exitProcess(0)
         }
         val start = System.currentTimeMillis()
+
+        // Parse the .ktlintignore file
+        parseKtlintIgnore(ktlintIgnorePath ?: ".")
```

Comment:
It would be useful to put support classes in the core module to locate any .ktlintignore files and parse them to disabled rules userdata. This would allow tools not invoking the CLI portion of ktlint to have an easy way to support .ktlintignore exactly the same way.

Reply:
sounds good. You actually know much more about the internals of `ktlint` than I do... 🤦‍♀